app/src/shortcutspage.cpp

```diff
@@ -117,6 +121,56 @@ void ShortcutsPage::keyCapLineEditTextChanged()
     qDebug() << "Shortcut " << strCmdName << " = " << strKeySeq;
 }
 
+void ShortcutsPage::saveShortcutsButtonClicked()
+{
+    QSettings settings( PENCIL2D, PENCIL2D );
+    QString fDir = settings.value("LastSavePath/Animation").toString();
```

Comment:
As far as I can see you never set this anywhere so the string will always be empty.

Reply:
It's set here I think: https://github.com/pencil2d/pencil/blob/220a27a06756bc524230211c7b80a9183eac6d23/app/src/filedialogex.cpp#L123-L125

Having said that, it definitely could still be empty, which should be checked so that it doesn't try to write to the root directory.

Reply:
Hmm that looks like a different value to me, I'd expect that what you posted should equal a string "Animation" not LastSavePath/Animation

Reply:
As I understand it, if you think of the LastSavePath/Animation, then beginGroup is like cd (or more accurately pushd).

Looks like it's set alright for me. if you're looking directly at the settings file, it will look like this:
```
[LastSavePath]
Animation=<some_path>
```

Take this with a grain of salt as I have not actually tested this branch yet.

Reply:
It works as expected from loadShortcutsButtonClicked but in saveShortcutsButtonClicked I never get a path. I've also debugged it a bit because the code is slightly different.

this in the load method will work:
```
    QSettings settings( PENCIL2D, PENCIL2D );
    QString fDir = settings.value("LastSavePath/Animation").toString();
```

While this in the save method will not.
```
settings.beginGroup( "LastSavePath" );
    QString fDir = settings.value("LastSavePath/Animation").toString();
```

If you however change the above to 
```
settings.beginGroup( "LastSavePath" );
QString fDir = settings.value("Animation").toString();
```

Then I get the expected behaviour.

Reply:
Maybe that's what is not working in Ubuntu, but is working on Win and iOS. As explained in another comment, I have som odd behavior on my Ubuntu. I'll test it when I get home.